src/codegen/compile_classical.ml

```diff
@@ -2540,7 +2540,7 @@ module Closure = struct
       I32Type :: Lib.List.make n_args I32Type,
       FakeMultiVal.ty (Lib.List.make n_res I32Type))) in
     (* get the table index *)
-    Tagged.load_forwarding_pointer env ^^
+    (*Tagged.load_forwarding_pointer env ^^ FIXME: NOT needed, accessing immut slots*)
```

Comment:
Did @luc-blaeser approve this? Seems a bit risky to me.

Reply:
Hmm, maybe that's ok because its the fun ptr field which is just an integer. Indeed, forward this would actually be wrong unless we represent those as heap object too now.

Reply:
I don't know how Luc wants to implement named (shared) functions, maybe then the index changes? Better ask!

Reply:
Please note that targets of immutable slots can also be moved by GC, so this  extra step is necessary I believe